steam: add page

thomas-m-d opened the pull request:
add page for valve's steam client

<!-- Thank you for sending a PR! -->
<!-- Please perform the following checks and mark all the boxes accordingly. -->
<!-- You can remove the checklist items that don't apply to your PR. -->

- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

marchersimon commented on pages/common/steam.md:
```diff
@@ -0,0 +1,12 @@
+
+> Video game platform by Valve.
+> More information: <https://store.steampowered.com/>.
+
+- Launch Steam client in background:
+
+`steam &`
```
```suggestion
```
Same as for the other page. It isn't really command specific. 

However I found <https://developer.valvesoftware.com/wiki/Command_Line_Options#Command-line_parameters_3>, which has a good amount of command line options for steam. It says it's for Windows, but I tested some and they all work for Linux too.

marchersimon replied:
If you like to, go ahead. I'll convert the PR to a draft in between.

thomas-m-d replied:
sounds good. hopefully i should have it done later today, but in the next couple days at the latest.

thomas-m-d commented:
ok, i added what i considered the essential examples, and deleted the `steam &` example, so the page is at 5 examples total. there were a couple more i could add to get to 8, but i'm assuming the goal for tldr is to use as few examples as possible while still covering the essentials?

marchersimon commented on pages/common/steam.md:
```diff
@@ -0,0 +1,24 @@
+> Video game platform by Valve.
+> More information: <https://developer.valvesoftware.com/wiki/Command_Line_Options#Steam_.28Windows.29>.
+
+- Launch Steam client with logs (incl. errors) appearing in terminal:
```
```suggestion
- Launch Steam and show debug messages in the terminal:
```

marchersimon commented on pages/common/steam.md:
```diff
@@ -0,0 +1,24 @@
+
+`steam`
+
+- Launch Steam client with a debug console tab included in-app:
```
```suggestion
- Launch Steam with a debug console tab included in-app:
```

marchersimon commented on pages/common/steam.md:
```diff
@@ -0,0 +1,24 @@
+
+`steam -console`
+
+- Log into Steam client with specified credentials:
```
```suggestion
- Log into Steam with the specified credentials:
```

marchersimon commented on pages/common/steam.md:
```diff
@@ -0,0 +1,24 @@
+
+`steam -tenfoot`
+
+- Shut down Steam client:
```
```suggestion
- Exit Steam:
```

owenvoke reviewed (approved):
LGTM!

marchersimon reviewed (changes requested):
> i'm assuming the goal for tldr is to use as few examples as possible while still covering the essentials?

You're absolutely right. Looks good to me so far, I just left a few suggestions below.

thomas-m-d commented:
anything i need to do on my end to get this merged? looks like no?

bl-ue commented:
No, you're good.  I'll take a look and then merge it 👍🏻

bl-ue commented on pages/common/steam.md:
```diff
@@ -0,0 +1,24 @@
+
+`steam`
+
+- Launch Steam with a debug console tab included in-app:
```
This is still correct, is it not?
```suggestion
- Launch Steam and enable the debug console tab:
```

thomas-m-d replied:
yes. however, i want to make it clear that the debug console is a tab in the steam app, to contrast it with the earlier example that gives you debug info in the bash shell you ran the command from. does something like:

`Launch Steam and enable its in-app debug console tab:`

look ok to you?

thomas-m-d replied:
committed this and the previous change

bl-ue commented on pages/common/steam.md:
```diff
@@ -0,0 +1,24 @@
+
+`steam -console`
+
+- Log into Steam with the specified credentials:
```
Can "Steam" here be replaced with the appropriate URL (e.g. https://<!---->steam.com)?  Unless that's not correct.

marchersimon replied:
That would be <https://store.steampowered.com/>.

thomas-m-d replied:
this command lets you skip the login prompt for the steam client gui, not something in your browser. normally you'd launch steam, enter your login info, then be able to access your games. this lets you skip the middle step.

bl-ue replied:
Oh, I see.  Then let's leave it as-is. 👍🏻

thomas-m-d commented on pages/common/steam.md:
```diff
@@ -0,0 +1,24 @@
+> Video game platform by Valve.
+> More information: <https://developer.valvesoftware.com/wiki/Command_Line_Options#Steam_.28Windows.29>.
+
+- Launch Steam and show debug messages in the terminal:
```
committed this and the next change.

bl-ue commented:
Oh oops! I forgot about this one 😅
